ui/widgets/nuxeo-checkmark.js

```diff
@@ -111,11 +112,21 @@ import '../nuxeo-icons.js';
       };
     }
 
+    ready() {
+      super.ready();
+      this.setAttribute('role', 'checkbox');
+      this.setAttribute('aria-checked', false);
```

Comment:
I don't think we want this. I believe this will set `aria-checked="false"` which actually means, as a boolean, that it is `true`.

I don't think we need to do anything here, as `aria-checked` is already false by default and we already have an observer to handle it.

Reply:
`aria-checked="false"` is actually a string that needs to be present so AT can pick up its status.